third_party/ienn/src/ie_compilation.cpp

```diff
@@ -629,8 +723,27 @@ int32_t Compilation::AddFullyConnectedV1(const Operation& operation) {
   try {
     const uint32_t weights_index = operation.inputs[1];
     const uint32_t bias_index = operation.inputs[2];
-    AddConstant(weights_index, true);
-    AddConstant(bias_index, true);
+    if (index_op_map_.find(weights_index) == index_op_map_.end()) {
```

Comment:
Do you mean the weights have been added?

Reply:
Yes, just as the weights of conv. It will make sense for quant resnet50 model and we are going to support it.

Reply:
How about moving to AddConstant, return directly if the weights have existed.